src/components/cards/Port.astro

```diff
@@ -61,6 +61,7 @@ const { title, link, categories, color } = Astro.props;
 
     .port-header {
       @include utils.flex($gap: var(--space-sm));
+      flex-wrap: nowrap;
```

Comment:
Curious what the reason for this is?

Reply:
Prevents the port name being too long and forcing the icon down a level.